app/controllers/questions_controller.rb

```diff
@@ -0,0 +1,47 @@
+class QuestionsController < ApplicationController
+  before_action :find_test
+
+  rescue_from ActiveRecord::RecordNotFound,  with: :rescue_with_question_not_found
+
+  def index
+    @questions = @test.questions.order(created_at: :asc).pluck(:body)
+    render plain: @questions * "\n"
+  end
+
+  def show
+    @question = @test.questions.find(params[:id])
+  end
+
+  def new
+    @question = @test.questions.new
+  end
+
+  def create
+    @question = @test.questions.new(question_params)
+    if @question.save
+      render :show
```

Comment:
так будет правильно:
```suggestion
      redirect_to @question
```
подробней это будет разобрано на следущем уроке, не помню, были ли детали по поводу разницы `redirec_to` vs `render` в этом скринкасте, можешь посмотреть в документации